openmrs/apps/reports/GEORGETOWN_PATIENT_INFORMATION_REPORT/sql/georgetownPatientInformationReport.sql

```diff
@@ -9,11 +9,15 @@ SELECT
     getHivTestDate(p.patient_id, "2000-01-01", "2100-01-01") as "hivTestDate",
     getProgramAttributeDateValueFromAttributeAndProgramName(p.patient_id, "PROGRAM_MANAGEMENT_2_PATIENT_TREATMENT_DATE", "HIV_PROGRAM_KEY") as "dateArvInitiation",
     CONCAT(getPatientVillage(p.patient_id),",",getPatientPreciseLocation(p.patient_id)) as "address",
+    getPatientRegion(p.patient_id) as "region",
+    getPatientDivision(p.patient_id) as "division",
+    getPatientSubDivision(p.patient_id) as "subDivision",
+    getPatientVillage(p.patient_id) as "village",
     getPatientPhoneNumber(p.patient_id) as "contactTelephone",
     getPatientMostRecentProgramTrackingStateValue(p.patient_id,"en","HIV_PROGRAM_KEY") as "clinicalWhoStage",
     getFirstARVPrescribed(p.patient_id) as "regimentAtArtInitiation",
     getARVTherapeuticLineAtInitiation(p.patient_id) as "lineAtInitiation",
-    getActiveARVWithLowestDispensationPeriod(p.patient_id, "2000-01-01", "2100-01-01") as "currentRegimen",
+    concatenateARTDrugs(p.patient_id, "2000-01-01", "2100-01-01")as "currentRegimen",
```

Comment:
@tresorluz If JP agrees with the definition I asked about in the slack chat, we'll need to plug in here the report start date and end date variables. I have included the comment down here too for easy reference:

is the following definition of “Current ARV Drugs” correct?
```ARV drugs dispensed to patients include those with treatments starting within the reporting period, as well as treatments starting prior to the reporting period BUT ending either during or after the reporting period. ARV drugs that started after the reporting period will not be included. Also, ARV that both started and ended before the reporting period will be excluded.```

Reply:
JP didn't agree with this